app/ui-utils/client/lib/reactionListContent.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box, Tag, Divider } from '@rocket.chat/fuselage';
+
+import { useTranslation } from '../../../../client/contexts/TranslationContext';
+import Emoji from '../../../../client/components/basic/Emoji';
+
+export function Reactions(props) {
+	const styleDivider = {
+		marginTop: '8px',
+		marginBottom: '16px',
+	};
+
+	const reactionKeys = Object.keys(props.reactions);
+
+	return <Box>
+		{
+			reactionKeys.map((reaction, i) => <Box key={reaction}><div style={{ display: 'flex' }}><Emoji emojiHandle={reaction}/> {reaction}</div>
+				<Usernames usernames={props.reactions[reaction].usernames}></Usernames> { i !== (reactionKeys.length - 1) && <Divider key={reaction} style={styleDivider}/> } </Box>)
+		}
+	</Box>;
+}
+
+export function Usernames(props) {
+	const styleBox = {
+		paddingTop: '8px',
+	};
+
+	const styleTag = {
+		marginRight: '4px',
```

Comment:
Prefer the [logical property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties):

```suggestion
		marginInlineEnd: '4px',
```